pkg/generators/config.go

```diff
@@ -0,0 +1,123 @@
+package generators
+
+import (
+	"github.com/nais/naiserator/pkg/naiserator/config"
+)
+
+// Options defines customizations for resource objects.
+// FIXME: clean up
+type Options struct {
+	// FIXME: digdirator hosts - in use or not?
```

Comment:
Not really in use anymore after the removal of Istio (and `ServiceEntry`), can be removed.